### DISCUSSION TITLE
add ai to elementary os

### DISCUSSION THREAD

**Discussion by Idan-A** [elementary/triage#545] [Ideas]
[llma3](https://huggingface.co/spaces/Be-Bo/llama-3-chatbot_70b)
[dalle](https://huggingface.co/spaces/mukaist/DALLE-4K) 
use for exemple these apis inside: mail,code or image gallery 
the problem is there aren't local and use cloud but if eOS is the first distro use ai inside the os it it could be a great advertisement for the team e for the distro

**Comment by danirabbit** [comment]
Over the last couple of years we've seen how AI—specifically large language models—has impacted our industry and so far the impact has been quite negative. LLMs have proved unreliable and have hurt software quality, even for seemingly simple tasks like providing summaries. They've also had a very negative impact on our environment and water use for data centers has disproportionally impacted marginalized communities. I'm afraid that even if AI made our products better to use, the external effects it has would go against our company mission and values.

There may yet be use cases for small, local machine learning models trained on ethically sourced data sets for things like improving photo search, and it's possible we maybe be able to incorporate these kinds of features in a way that aligns with our values and reliably improves our operating system.

Until that time, I think I have to close this idea as a "design conflict" sorry! Thanks for proposing your idea :)
